conf/base.config

```diff
@@ -14,9 +14,9 @@ process {
     memory = { check_max( 7.GB * task.attempt, 'memory' ) }
     time = { check_max( 4.h * task.attempt, 'time' ) }
 
-    errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }
-    maxRetries = 3
-    maxErrors = '-1'
+    errorStrategy = { task.exitStatus in ((130..145) + 104) ? 'retry' : 'finish' }
+    maxRetries    = 1
```

Comment:
Where is this coming from?

Reply:
This is coming from the template: https://github.com/nf-core/metapep/blob/TEMPLATE/conf/base.config#L14

Reply:
The one retry is in the template for a long time, nf-core/mag also kept their 3 retry strategy. Should we also stay with 3 retries?

Reply:
then one would explicitly set the label `error_retry` to processes where one would want to allow retrying, I would tend to do that at last for the actual prediction processes, since there are so many tasks run and the probablitiliy that something goes wrong seems high ...